### ISSUE TITLE
copr build

### ISSUE THREAD

**Issue by stsp** [jwt27/build-gcc#18]
Hi.

I am trying out copr for my packages:
https://copr.fedorainfracloud.org/
Would you like to deploy the djgpp there?

**Comment by jwt27** [comment]
For Fedora and RH, rpms are already available from the djgpp ftp server:
ftp://ftp.delorie.com/pub/djgpp/rpms/

**Comment by stsp** [comment]
Yes, but aren't they lack your/mine
patches?
And the copr faq says:
```
Can I depend on other packages, which are not in Fedora/EPEL?
Yes, they just need to be available in some yum repository.
```
So it doesn't work with plain rpms.
I really think that deploying an online
build and using your repo as an
upstream is a good idea, as the original
djgpp upstream is kinda hopeless.

**Comment by stsp** [comment]
If you don't want to register anywhere,
I can deploy it myself. What is needed
is to make sure "rpkg local" works locally.
See how I converted a few packages today:
https://github.com/dosemu2/comcom32/commit/4b525a7cbc645f68d0d413841b562e63474a8bc7
Its really not difficult.

**Comment by jwt27** [comment]
Right, those patches are still an issue., I think you should ping those on the mailing list again and ask why they haven't been included yet. Maybe they just forgot.
When I have time I'll set up a Fedora VM and look into copr.

**Comment by stsp** [comment]
I deployed the copr build here:
https://copr.fedorainfracloud.org/coprs/stsp/dosemu2/package/build-gcc/
So you can derive from that.
